.solcover.js

```diff
@@ -2,23 +2,23 @@ const fs = require('fs');
 const path = require('path');
 const mv = require('mv');
 
-const v4ArtifactsPath = path.join(__dirname, '.coverageV4Artifacts');
+const oldArtifactsPath = path.join(__dirname, '.coverageArtifacts');
```

Comment:
I don't follow what does old mean here?
can it be named previous?
or is it old?